src/options/Keybindings/default.json

```diff
@@ -654,6 +654,123 @@
         "clipboardNewWindow": [],
         "clipboardIncognitoWindow": []
       }
+    },
+    {
+      "name": "VimFx",
+      "options": {
+        "physicalKeys": false,
+        "ignoreModifierKeys": false,
+        "toggleHelpMenu": [[{ "code": "Slash", "key": "?", "shiftKey": true }]],
+        "toggleSaka": [],
+        "openLink": [[{ "code": "KeyF", "key": "f" }]],
+        "openLinkInBackgroundTab": [
+          [{ "code": "KeyF", "key": "F", "shiftKey": true }]
+        ],
+        "openLinkInForegroundTab": [{ "code": "KeyE", "key": "e"},{ "code": "KeyT", "key": "t"}],
+        "openLinkInNewWindow": [{ "code": "KeyE", "key": "e"},{ "code": "KeyW", "key": "w"}],
+        "openLinkInIncognitoWindow": [{ "code": "KeyE", "key": "e"},{ "code": "KeyP", "key": "p"}],
+        "downloadLink": [],
+        "focusLink": [],
+        "focusInput": [
+		  [{ "code": "KeyE", "key": "e" }, { "code": "KeyF", "key": "f" }]
+        ],
+        "hintChars": "sadfjklewcmpgh",
+        "scrollDown": [
+          [{ "code": "KeyJ", "key": "j" }]
+        ],
+        "scrollUp": [
+          [{ "code": "KeyK", "key": "k" }]
+        ],
+        "scrollLeft": [[{ "code": "KeyH", "key": "h" }]],
+        "scrollRight": [[{ "code": "KeyL", "key": "l" }]],
+        "scrollPageDown": [[{ "code": "Space", "key": "Space" }]],
+        "scrollPageUp": [[{ "code": "Space", "key": "Space", "shiftKey": true }]],
+        "scrollHalfPageDown": [[{ "code": "KeyD", "key": "d" }]],
+        "scrollHalfPageUp": [[{ "code": "KeyU", "key": "u" }]],
+        "scrollToBottom": [[{ "code": "KeyG", "key": "G", "shiftKey": true }]],
+        "scrollToTop": [
+          [{ "code": "KeyG", "key": "g" }, { "code": "KeyG", "key": "g" }]
+        ],
+        "scrollToLeft": [],
+        "scrollToRight": [],
+        "goBack": [[{ "code": "KeyH", "key": "H", "shiftKey": true }]],
+        "goForward": [[{ "code": "KeyL", "key": "L", "shiftKey": true }]],
+        "nextPage": [],
```

Comment:
What shortcut in Saka Key is this? It _sounds_ a lot like VimFx's `follow_next` command, but as far as I know Saka Key doesn't have such a command. Confusing. Anyway, the default shortcuts for clicking next/previous links in VimFx are `[`/`]`.

Reply:
This would be "Go backward/forward in history", which is H/L in VimFx. I don't now what "follow_next" does in VimFx, actually. If it's not in Saka Key, I can't add it anyway.

Reply:
Ok, nothing to do here then :)